.github/workflows/go.yml

```diff
@@ -94,3 +94,35 @@ jobs:
       - name: Test
         if: runner.os == 'Windows'
         run:  .ci/scripts/test.bat
+
+  test-freebsd:
+    runs-on: ubuntu-latest
+    env:
+      # Only GITHUB_* are passed into the VM.
+      GITHUB_GOLANG_VERSION: 1.22.1
+    name: test (1.22.1, freebsd-14.0, cgo)
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Test
+        # TODO: Skip until freebsd provider is merged. Then this 'if' should be removed.
+        if: ${{ hashFiles('providers/freebsd/') != '' }}
+        id: test
+        uses: vmactions/freebsd-vm@35a5b20a98476a681c7576a344775be7e7f77f06  #v1.0.6
+        with:
+          usesh: true
+          prepare: |
+            freebsd-version
+            pwd
+            ls -lah
+            whoami
+
```

Comment:
required?

Reply:
Required, no. But I didn't see any downside to not having it. There's already a ton of logging happening with the freebsd plugin so this is like a drop in the bucket.